daemon/network.go

```diff
@@ -318,8 +318,8 @@ func (daemon *Daemon) DisconnectContainerFromNetwork(containerName string, netwo
 
 // GetNetworkDriverList returns the list of plugins drivers
 // registered for network.
-func (daemon *Daemon) GetNetworkDriverList() map[string]bool {
-	pluginList := make(map[string]bool)
+func (daemon *Daemon) GetNetworkDriverList() []string {
+	pluginList := []string{}
```

Comment:
nit: `var pluginList []string` looks more go-ish to me, but it is just my opinion
https://github.com/golang/lint/blob/c7bacac2b21ca01afa1dee0acf64df3ce047c28f/lint.go#L1311